src/main/java/com/amihaiemil/camel/WellIndentedLine.java

```diff
@@ -97,4 +97,17 @@ public int indentation() {
     public String toString() {
         return this.line.toString();
     }
+
+    @Override
+    public int compareTo(final YamlLine other) {    
+        int result = -1;
```

Comment:
In this method, just ``return this.line.compareTo(other)``, since the class is a decorator (same as EvenlyIndentedLine)